src/components/AppNavigation/AppNavigation.vue

```diff
@@ -65,7 +65,9 @@
       {{ $t("filter") }}
 
       <div class="app-menu__filter-indicator">
-        {{ spacesStore.numberOfSelectedFilters }}
+        <ClientOnly fallback="...">
+          {{ spacesStore.numberOfSelectedFilters }}
+        </ClientOnly>
```

Comment:
Unrelated to changes but fixes some hydration errors, I disagree with https://github.com/voorhoede/tudelft-spacefinder/blob/master/docs/decision-log/2023-05-02-not-fixing-hydration-warnings.md ;)